R/extract_eq.R

```diff
@@ -91,11 +94,15 @@
 
 extract_eq <- function(model, intercept = "alpha", greek = "beta",
                        raw_tex = FALSE, ital_vars = FALSE,
+                       show_distribution = FALSE,
                        wrap = FALSE, terms_per_line = 4,
                        operator_location = "end", align_env = "aligned",
                        use_coefs = FALSE, coef_digits = 2, fix_signs = TRUE) {
+  if(show_distribution) {
```

Comment:
This will need to change slightly with 8eb90c9 - there, I split wrapping and the align environment into two separate concepts. When `wrap = TRUE`, the LHS will get split into `terms_per_line` chunks. It also triggers the aligned environment.

Equations with multiple equations, like ordered logit/probit, also trigger the aligned environment (based on the length of `rhs_combined`). But it's also possible to have multi-line equations that *aren't* wrapped, hence the splitting of the logic

Thus, any equation with multiple lines will automatically get put in `aligned`, regardless of wrapping status. Any equation with wrapping enabled will also get put in `aligned`

Reply:
That sounds great! I was running into issues with it when I had `wrap = FALSE`. I think I already had https://github.com/datalorax/equatiomatic/commit/8eb90c95ee4cc51b1a34ddc7be23030d8b3f4ec2 merged in so we may need to look at this closer.